src/NzbDrone.Core/NetImport/RSSImport/RSSImport.cs

```diff
@@ -12,6 +12,7 @@ public class RSSImport : HttpNetImportBase<RSSImportSettings>
         public override string Name => "RSSList";
         public override bool Enabled => true;
         public override bool EnableAuto => false;
+		public override bool ShouldClean => false;
```

Comment:
We should have a property similar to indexers called CanClean. This way the switch can be disabled in the ui, as to not confuse users why their tmdb list is not getting cleaned.